### PR TITLE
update GAPIC clients

### PR DESCRIPTION
**Pull request by pongad**

### CODE REVIEW

File: google-api-grpc/pom.xml [hunk 1]
```diff
@@ -446,6 +456,16 @@
         <artifactId>grpc-google-cloud-vision-v1p2beta1</artifactId>
         <version>1.18.1-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-vision-v1p2beta1:current} -->
       </dependency>
+      <dependency>
```
Comment (garrettjonesgoogle) [inline google-api-grpc/pom.xml]: Comment out this block too

File: google-cloud-bom/pom.xml [hunk 2]
```diff
@@ -220,6 +220,21 @@
       </dependency>
 
       <!-- API-specific dependencies -->
+      <dependency>
```
Comment (garrettjonesgoogle) [inline google-cloud-bom/pom.xml]: Stuff to comment out in this file

File: versions.txt [hunk 3]
```diff
@@ -45,6 +45,7 @@ grpc-google-cloud-video-intelligence-v1p1beta1:0.19.0:0.19.1-SNAPSHOT
 grpc-google-cloud-vision-v1:1.18.0:1.18.1-SNAPSHOT
 grpc-google-cloud-vision-v1p1beta1:0.19.0:0.19.1-SNAPSHOT
 grpc-google-cloud-vision-v1p2beta1:1.18.0:1.18.1-SNAPSHOT
+grpc-google-cloud-vision-v1p3beta1:0.19.0:0.19.1-SNAPSHOT
```
Comment (garrettjonesgoogle) [inline versions.txt]: comment out or revert file

File: google-api-grpc/pom.xml (hunk 1 above)
Comment (pongad) [reply google-api-grpc/pom.xml]: Done

File: google-cloud-bom/pom.xml (hunk 2 above)
Comment (pongad) [reply google-cloud-bom/pom.xml]: Done

**Pull request merged**